packages/core/src/generators/angular.ts

```diff
@@ -276,6 +271,42 @@ export const componentToAngular: TranspilerGenerator<ToAngularOptions> =
     const outputVars = uniq([...metaOutputVars, ...getPropFunctions(json)]);
     const stateVars = Object.keys(json?.state || {});
 
+    const DEFAULT_OPTIONS: ToAngularOptions = {
+      preserveImports: false,
+      preserveFileExtensions: false,
+      importMapper: (_component: any, theImport: any, importedValues: any, componentsUsed: any) => {
+        let importPath = theImport.path;
+
+        const { defaultImport, namedImports, starImport } = importedValues;
+
+        let importValue;
+
+        if (starImport) {
+          importValue = ` * as ${starImport} `;
+        } else {
+          importValue = [defaultImport, namedImports]
+            .filter(Boolean)
+            .map((importName) => {
+              if (!userOptions.standalone) {
+                for (const usedComponentName of componentsUsed || []) {
+                  if ((importName || '').indexOf(usedComponentName) > -1) {
+                    // replace the component name with the module name
+                    return (importName || '').replace(
+                      usedComponentName,
+                      `${usedComponentName}Module`,
+                    );
+                  }
+                }
+              }
+              return importName;
+            })
+            .join(', ');
+        }
+
+        return `import ${importValue} from '${importPath}';`;
```

Comment:
I think you're missing wrapping brackets `{ }`, right now the PR imports the same default output, just under a different name. I don't think this is what you want?

Is this what's needed instead?

```suggestion
        return `import {${importValue}} from '${importPath}';`;
```